internal/pkg/model/v2/activity/service/js.go

```diff
@@ -110,13 +108,6 @@ func NewVM(defaults map[string]interface{}) (vm *VM, err error) {
 			vm.vm.Set(k, v)
 		}
 	}
-	// Add ENV to the VM
```

Comment:
This might still need to be here for conditional evaluations with 'env' in them to work.

Reply:
Got it, thanks.